app/scripts/controller-init/notifications/notification-services-push-controller-init.ts

```diff
@@ -36,7 +36,7 @@ export const NotificationServicesPushControllerInit: ControllerInitFunction<
       vapidKey: process.env.VAPID_KEY ?? '',
     },
     config: {
-      isPushFeatureEnabled: isManifestV3,
+      isPushFeatureEnabled: isManifestV3 && !process.env.IN_TEST,
```

Comment:
Lol, push notifications are called now, so are causing e2e failures.

We are introducing a new v2 architecture revamp, so will re-enable push notifications for tests when that is in.
(the e2e test still run without push notifications)